lib/tasks/scheduler.rake

```diff
@@ -11,8 +11,6 @@ task house_valuation_collector: :environment do
       Sentry.capture_exception(e, extra: { house_id: house.id })
     end
   end
-
-  HappyHood::Slack::Client.send_daily_price_summary
```

Comment:
You can get rid of the `require` for the slack client

Reply:
Updated!